lib/on_membership_event.js

```diff
@@ -137,10 +142,12 @@ function createUpdatedHandlerForRing(ringpop) {
             var update = updates[i];
             switch (update.status) {
                 case Member.Status.alive:
+                // TODO: Faulty should be here
```

Comment:
I think I might just go ahead and add it, I think @thanodnl added it in ringpop-go; it's also the correct behaviour.